documenteer/bin/refreshlsstbib.py

```diff
@@ -0,0 +1,119 @@
+"""Command line executable to refresh lsst bib files.
+"""
+
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+from builtins import *  # NOQA
```

Comment:
Do you really need this? How many builtins are you relying on future to provide in python2?